core/src/navigation/LeftNav.html

```diff
@@ -291,4 +305,12 @@
       width: 1.25rem;
     }
   }
+
+  .lui-sideNavFooter {
+    position: sticky;
```

Comment:
`fixed` instead of `sticky`? It seems to be placed not at the end of the navbar.
<img width="331" alt="screen markus e 2019-04-30 at 15 10 07" src="https://user-images.githubusercontent.com/1720843/56963830-2f853b00-6b5a-11e9-99df-50774cd36a32.png">

Reply:
Reading the conversation on Slack about this, I understood it as `sticky`.

Reply:
... but looking at the Zeplin, it's fixed. I'm confused 😕